apps/native/app/src/stores/preferences-store.ts

```diff
@@ -49,6 +56,13 @@ const defaultPreferences = {
   hasAcceptedBiometrics: false,
   hasOnboardedPasskeys: false,
   hasCreatedPasskey: false,
+  homeScreenEnableGraphicWidget: true,
+  homeScreenEnableInboxWidget: true,
+  homeScreenEnableApplicationsWidget: true,
+  homeScreenEnableLicensesWidget: true,
+  homeScreenEnableVehiclesWidget: true,
+  homeScreenEnableAirDiscountWidget: true,
+  homeScreenWidgetsInitialised: false,
```

Comment:
I thought a lot about having these shorter. But I struggled to find something descriptive enough, ideas welcome 😄

Reply:
Skip the homeScreen prefix? This is hard

```
graphicWidgetEnabled: true,
inboxWidgetEnabled: true,
applicationsWidgetEnabled: true,
licensesWidgetEnabled: true,
vehiclesWidgetEnabled: true,
airDiscountWidgetEnabled: true,
widgetsInitialized: false,

```